### ISSUE TITLE
Are Spies working?

### ISSUE THREAD

**Issue by Infixo** [Infixo/Civ6-Real-Strategy#12]
[from Steam workshop]
beesechurger 18 Mar, 2021 @ 4:03pm 
A lot of people seem to claim that this mod is now broken due to the updates, I haven't really noticed anything, but one thing I have noticed just now that kinda made me pissed off is that I can't remember the last time the AI leaders sent spies to my cities.. like.. it has not happened one single game for a loong time. I think this mod might be the culprit, anyone else encounter this?

**Comment by Phyrax1** [comment]
Yes they do. AI spies are very annoying and keeps neutralizing governors.

**Comment by Infixo** [comment]
That is also my observation. They do neutralize govs, steal money and sometimes raise partisans. I remember they loved to raise partisans at the begining, like always. Anyway, I regularly catch their spies.
However, RST doesn't influence how spies are used, only if produced. Need to review the params here.
